components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -566,12 +654,26 @@ bool AdsServiceImpl::IsAdsEnabled() const {
 
 void AdsServiceImpl::set_ads_enabled(bool enabled) {
   profile_->GetPrefs()->SetBoolean(prefs::kBraveAdsEnabled, enabled);
+  // Once ads has been enabled once, we should no longer
+  // show the launch notification.
+  if (enabled) {
```

Comment:
nit so we fail early and reduce what could eventually become the pyramid of doom:

```
if (!enabled) {
  return;
}
```

Reply:
done